Upstream: Not applicable
Reason: relax dependency requirements as our version of cryptography
 even if it is old does not bundle a vulnerable 'openssl'

diff --git a/synapse/python_dependencies.py b/synapse/python_dependencies.py
index 989523c..19a4167 100644
--- a/synapse/python_dependencies.py
+++ b/synapse/python_dependencies.py
@@ -86,7 +86,7 @@ REQUIREMENTS = [
     "typing-extensions>=3.7.4",
     # We enforce that we have a `cryptography` version that bundles an `openssl`
     # with the latest security patches.
-    "cryptography>=3.4.7",
+    "cryptography",
 ]
 
 CONDITIONAL_REQUIREMENTS = {
